firefox_puppeteer/base.py

```diff
@@ -22,3 +23,21 @@ def marionette(self):
 
     def get_marionette(self):
         return self.marionette
+
+
+class UIBaseLib(BaseLib):
+    """A base class for all UI element wrapper classes inside a chrome window."""
+
+    def __init__(self, marionette_getter, window):
+        BaseLib.__init__(self, marionette_getter)
+
+        # importing globally doesn't work
+        from .ui.windows import BaseWindow
```

Comment:
Ok, I can move it to the top-level but I will have to import it with the full absolute path. This would solve at least your concern here. We can check that sometime later.